apps/condo/domains/news/constants/common.js

```diff
@@ -4,7 +4,7 @@ const conf = require('@open-condo/config')
 
 const SENDING_DELAY_SEC = Number(get(conf, 'NEWS_ITEMS_SENDING_DELAY_SEC', 15))
 const NEWS_SENDING_TTL_IN_SEC = Number(get(conf, 'NEWS_ITEM_SENDING_TTL_SEC', 60 * 30))
-const LOAD_RESIDENTS_CHUNK_SIZE = 20
+const LOAD_RESIDENTS_CHUNK_SIZE = 50
```

Comment:
We decrease this const from 50 to 20 before. For fix out of memory errors when fetch residents?
If yes, are you sure that now it's will be work fine?